airline/models.py

```diff
@@ -1,8 +1,10 @@
 from django.db import models
 from cities_light.models import Country
+from cities_light.models import City
 class Airline(models.Model):
     name = models.CharField(max_length=255)
     country = models.ForeignKey(Country, on_delete=models.CASCADE)
+    city = models.ForeignKey(City, on_delete=models.CASCADE)
```

Comment:
Hey @KyleJYKim regarding the `airline` model I think, as Anwar had mentioned, it is ok to leave the country it belongs to but having the city maybe it is too detailed information.
So we can remove that new property.